diagnostic.js

```diff
@@ -29,21 +29,39 @@ mongoose.Promise = global.Promise;
 mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
+const controller = require('lib/wiring/controller');
+const models = require('app/models');
+const Art = models.art;
```

Comment:
These two things are constructs from Antony's Express template - they are _not_ part of Mongoose and do not apply here.